app/services/acuant/acuant_client.rb

```diff
@@ -20,6 +21,18 @@ def post_back_image(image:, instance_id:)
       ).fetch
     end
 
+    def post_images(front_image:, back_image:, instance_id:)
+      front_response = post_front_image(image: front_image, instance_id: instance_id)
```

Comment:
Don't we also need the create document and get results requests here? I think if we add that we can drop the `instance_id` from the args

Reply:
I just mirrored the methods that were already there. We call create_document in the step, get the instance_id and save that in the flow_session. I guess we don't need to do any of that now that we are doing it all in one call?

Reply:
OK, I tried creating the document here, but we need to save it in the `flow_session` so it's available elsewhere and `flow_session` isn't available from here.